math/libqalculate/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        qalculate libqalculate 3.3.0 v
+github.setup        qalculate libqalculate 3.12.0 v
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Also, the `epoch` may need to be added and set to 1 because "12" is alphabetically less than "3".